native/src/components/PoiDetails.tsx

```diff
@@ -47,15 +50,16 @@ type PoiDetailsProps = {
   poi: PoiModel
   language: string
   distance: number | null
+  onFocus?: () => void
```

Comment:
🤔 Does this need to be optional? Same for `PoiListItem`
```suggestion
  onFocus: () => void
```